src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
It's a bit weird to have a different name for the plugin that the real name. Not sure what to put :/